packages-native/intro-screen/src/IntroScreen.xml

```diff
@@ -0,0 +1,131 @@
+    </icon>
+    <properties>
+        <propertyGroup caption="General">
+            <propertyGroup caption="Content">
+                <property key="slides" type="object" isList="true">
+                    <caption>Slides</caption>
+                    <description>Each slide will add a drop zone.</description>
+                    <properties>
+                        <propertyGroup caption="Slide">
+                            <property key="name" type="string">
+                                <caption>Name</caption>
+                                <description />
+                            </property>
+                            <property key="content" type="widgets">
+                                <caption>Content</caption>
+                                <description/>
+                            </property>
+                        </propertyGroup>
+                    </properties>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Display">
+                <property key="showMode" type="enumeration" defaultValue="fullscreen">
+                    <caption>Show as</caption>
+                    <description />
+                    <enumerationValues>
+                        <enumerationValue key="fullscreen">Fullscreen</enumerationValue>
+                        <enumerationValue key="popup">Popup</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="buttonPattern" type="enumeration" defaultValue="all">
+                    <caption>Buttons</caption>
+                    <description>The caption and icon of the buttons change dynamically based on which slide the user is. For example on the first page a skip button is shown but now a previous button.</description>
+                    <enumerationValues>
+                        <enumerationValue key="all">Skip / Previous and Next / Done</enumerationValue>
+                        <enumerationValue key="next_done">Only Next / Done</enumerationValue>
```

Comment:
use of underscore not recommended, use camelCase if possible `nextDone`

Reply:
Done